packages/client/hmi-client/src/components/model/petrinet/tera-model-equation.vue

```diff
@@ -40,8 +40,6 @@ const props = defineProps<{
 	model: Model;
 }>();
 
-const emit = defineEmits(['model-updated']);
```

Comment:
intentional?

Reply:
Yes, it was not used and it create issue on Github PRs

Reply:
👍